exec/cnex/cnex.c

```diff
@@ -754,14 +754,26 @@ void exec_GEY(TExecutor*self)
     push(self->stack, r);
 }
 
-void exec_EQA(void)
+void exec_EQA(TExecutor*self)
```

Comment:
Why isn't there a space either before or after the `*`? This isn't multiplication.

Reply:
Found and fixed several locations where (for some reason) I didn't have a space between these.  Fixed.